types/zipObj.d.ts

```diff
@@ -1,3 +1,14 @@
-export function zipObj<K extends PropertyKey>(keys: readonly K[]): <T>(values: readonly T[]) => { [P in K]: T };
-export function zipObj<T, K extends PropertyKey>(keys: readonly K[], values: readonly T[]): { [P in K]: T };
+import * as _ from 'ts-toolbelt';
 
+export type _ZipObj<K extends readonly PropertyKey[], V extends {[key in keyof K]: unknown} | readonly unknown[]> =
```

Comment:
This utility type cannot live in this file. The build script combines all the files into one and copies the `ramda` block comments over. It forces us to have one export function per file

Don't add it to `src/types/util/tools.d.ts` though. I want to break that file up into a bunch of smaller ones (who has the time, right?)

Instead, do it like I have `src/types/util/deepModify.d.ts` by creating a new file `src/types/util/zipObj.d.ts`.